website/pages/docs/job-specification/template.mdx

```diff
@@ -59,7 +59,10 @@ README][ct]. Since Nomad v0.6.0, templates can be read as environment variables.
   templates, but we recommend using `source` for larger templates.
 
 - `destination` `(string: <required>)` - Specifies the location where the
-  resulting template should be rendered, relative to the task directory.
+  resulting template should be rendered, relative to the [task working
+  directory]. Only drivers that build a chroot into the task working directory
+  (such as `exec` or `raw_exec`) can render templates outside of the
```

Comment:
This implies that raw_exec makes a whole task chroot. I thought that the lack of isolation provides the template the "outside" of the typical Nomad paths feel? (maybe just a nitpick?)

Reply:
No you're right, that reads misleadingly.

Reply:
Fixed in daaf9ca